org.gnome.SimpleScan.json

```diff
@@ -250,6 +250,13 @@
                         "type": "git",
                         "tag-pattern": "^([\\d.]+)$"
                     }
+                },
+                {
+                    "type": "patch",
+                    "paths": [
+                        "simple-scan-Use-RDNN-app-ID.patch"
+                    ],
+                    "use-git": true
```

Comment:
```suggestion
                    "path": "simple-scan-Use-RDNN-app-ID.patch"
```

Reply:
Isn't that functionally equivalent?

Reply:
less lines is better. if you don't need it don't put it

Reply:
I don't agree but it's not worth arguing about! I pushed a new version.